### DISCUSSION TITLE
semantic segmeation-aireal images

### DISCUSSION THREAD

**Discussion by itaim18** [roboflow/notebooks#350] [Q&A]
Hey, I got a roboflow dataset with bunch of semantic segmented labels and classes, how should I go around training my model from here.
Was looking at https://github.com/roboflow/notebooks/blob/main/notebooks/fine-tune-sam-2.1.ipynb but not entirely sure if this is ther right approach
any ideas/

**Comment by LinasKo** [comment]
Hi @itaim18 👋 

I'd recommend simpler models if you're just starting off. Yolov8 and Yolo11 are the most popular ones, with little difference between them.

My favourite method is uploading to Roboflow app (app.roboflow.com), creating a dataset version and training there, but you can also train locally.

The two guides I think are helpful:
* https://blog.roboflow.com/train-yolov11-instance-segmentation/
* https://blog.roboflow.com/train-yolov11-instance-segmentation/#train-yolo11-on-a-custom-dataset

Hope it helps!
